app/data_providers/snowplow_corpus_slate_tracker.py

```diff
@@ -0,0 +1,103 @@
+import os
+from abc import ABC
+from typing import List, Any, Dict, Optional
+
+from aws_xray_sdk.core import xray_recorder
+from snowplow_tracker import Tracker, Subject, SelfDescribingJson, Emitter
+
+from app.models.corpus_slate_model import CorpusSlateModel
+from app.config import ENV, ENV_PROD
+
+
+class CorpusSlateTrackable(ABC):
```

Comment:
we have only one tracker for now, so probably don't need this abstract class, unless you need it for tests or smth